ci/tests/puppeteer/run.sh

```diff
@@ -235,13 +256,51 @@ if [[ "${REBUILD}" == "true" && "${RERUN}" != "true" ]]; then
   FLAGS=$(echo $BUILDFLAGS | sed 's/ //')
   printgreen "\nBuilding CAS found in $PWD for dependencies [${dependencies}] with flags [${FLAGS}]"
 
-  ./gradlew :webapp:cas-server-webapp-${project}:build \
-    -DskipNestedConfigMetadataGen=true -x check -x javadoc --build-cache --configure-on-demand --parallel \
-    ${BUILD_SCRIPT} ${DAEMON} -DcasModules="${dependencies}" -q ${BUILDFLAGS}
-
-  if [ $? -eq 1 ]; then
+  printcyan "Launching build in background to make observing slow builds easier"
+  targetArtifact=./webapp/cas-server-webapp-${project}/build/libs/cas-server-webapp-${project}-${casVersion}.${projectType}
+  if [[ -d ./webapp/cas-server-webapp-${project}/build/libs ]]; then
+    rm -rf ./webapp/cas-server-webapp-${project}/build/libs
+  fi
+  buildcmd=$(printf '%s' \
+      "./gradlew :webapp:cas-server-webapp-${project}:build \
+      -DskipNestedConfigMetadataGen=true -x check -x test -x javadoc --build-cache --configure-on-demand --parallel \
+      ${BUILD_SCRIPT} ${DAEMON} -DcasModules="${dependencies}" -PdependencyCacheDays=10 --no-watch-fs --max-workers=8 ${BUILDFLAGS}")
```

Comment:
Don't you think 10 is very very long? To minimize differences, I think we should make this number smaller, like 1.